easybuild/easyconfigs/e/EMU/EMU-0.66-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonPackage'
+
+name = 'EMU'
+version = '0.66'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.popgen.dk/software/index.php/EMU'
+
+description = """EMU infers population structure in the presence of missingness and works for both haploid, 
+psuedo-haploid and diploid genotype datasets
+"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['https://github.com']
+sources = ['Rosemeis/emu/archive/a6148ec9edefee19df0fcb6dc0a8272b9e5c96bc.zip']
+checksums = ['6e5166719fde2ec2a66bb25bc6ce6fc19f30b987081ebf1606e5f4ce4bdffe31']
```

Comment:
```suggestion
source_urls = ['https://github.com/Rosemeis/emu/archive/']
sources = ['a6148ec9edefee19df0fcb6dc0a8272b9e5c96bc.zip']
checksums = ['6e5166719fde2ec2a66bb25bc6ce6fc19f30b987081ebf1606e5f4ce4bdffe31']
```

Reply:
might possible also want to use the 
```
local_commit = 'a6148ec'
...
sources = [{'download_filename': '%s.tar.gz' % commit, 'filename': SOURCE_TAR_GZ}]
```
compare with XCfun: https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/x/XCFun/XCFun-20180122-intel-2017b-Python-2.7.14.eb